package.json

```diff
@@ -11,11 +11,15 @@
     "lint": "eslint ./src/index.js"
   },
   "dependencies": {
+    "clone": "^2.1.0"
+  },
+  "peerDependencies": {
```

Comment:
I figured we would want whoever is consuming the dependency to define it. In groveco, we explicitly define backbone and underscore. This is at the hope minimizing potential package duplication and to reduce bundle size. BUt to be fair this is extremely unlikely with our set up and might not be worth it to manage in this model

Reply:
the dev dependencies are just to install the peer dependencies when you are working on the module

Reply:
I'm bad at explaining these things sometimes, but what I'm really trying to prevent is something like [this](http://npm.github.io/npm-like-im-5/npm3/duplication.html). Last I checked this is still an issue with npm, but I've been a bit removed from it from working with yarn for a while.

Reply:
Looks like [it was fixed back in 2015](https://github.com/npm/npm/issues/6912)

Reply:
Gonna remove this then. It's more than likely unnecessary